iphone/metadata/en-US/release_notes.txt

```diff
@@ -1,8 +1,11 @@
-• New OSM maps data as of January 3, 2022
-• Improved search results ranking
-• Fixed hundreds of "capitals" on the World map
-• Fixed invalid inter-region routes
-• Search recognizes rd/st/ct=road/street/court synonyms
-• Consistent naming/translations for Bookmark Lists
-• Added Belarusian, Persian, incomplete Hebrew, and Swahili translations
-• Updated German, Italian, Romanian, Russian, Ukrainian, Turkish translations
+• New OSM maps data as of February 9, 2022
+• Fixed some cyclist routing bugs
+• Adjusted highways visibility
+• Transparent bottom toolbar background
+• New Help/About dialog on the main screen
+• Improved rendering of train stations, piers, dams
+• Improved search for fuel stations, posts, recycling and dump stations
+• Improved house numbers display in navigation mode
+• Made national park & military fills less obtrusive
+• Fixed some capitals incorrectly marked as cities on a World map
+• Updated ru, uk, tr translations
```

Comment:
```suggestion
• New OSM maps data as of February 4, 2022
• New Help/About dialog on the main screen
• Hide bottom toolbar on tap at the empty map
• Transparent bottom toolbar background

Routing:
• Fixed routing bugs with unnecessary exits from the highway
• Fixed some cyclist routing bugs
• Respect "Avoid roads" setting for bicycle and pedestrian routes
• Fixed bug with invalid cross-region routing

Map:
• Added search categories for Water, RV Facilities and Recycling
• Added parcel pickup, excrement bags, quay, marina, caravan, dump station, mast, flag pole, funeral directors
• Better displaying of house numbers in navigation
• Select linear objects instead of area ones on long tap
• Towns and cities are properly prioritized on 10-11 zoom levels
• Prioritize parking over other areas like forests
• Improved rendering of piers, dams, cutlines
• Improved train stations visibility
• Made national parks & military fills less obtrusive
• Adjusted highways visibility
• Fixed some capitals incorrectly marked as cities on a World map.
• Corrected some icons

Translations:
• Fixed German, Italian and Turkish translations
• FAQ in Turkish
```